logstash-filter-duration.gemspec

```diff
@@ -1,12 +1,10 @@
 Gem::Specification.new do |s|
-  s.name = 'logstash-filter-example'
-  s.version         = '3.0.0'
+  s.name = 'logstash-filter-duration'
+  s.version         = '0.1.0'
   s.licenses = ['Apache License (2.0)']
-  s.summary = "This example filter replaces the contents of the message field with the specified value."
+  s.summary = "This filter parses duration in ISO format."
   s.description     = "This gem is a Logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/logstash-plugin install gemname. This gem is not a stand-alone program"
-  s.authors = ["Elastic"]
-  s.email = 'info@elastic.co'
-  s.homepage = "http://www.elastic.co/guide/en/logstash/current/index.html"
```

Comment:
`s.homepage = 'https://github.com/funkwerk/logstash-duration-filter/'`
that people could find where the gem came from